packages/jmix-react-ui/src/ui/table/DataTableIntervalFunctions.test.ts

```diff
@@ -3,6 +3,8 @@ import dayjs from 'dayjs';
 import customParseFormat from 'dayjs/plugin/customParseFormat';
 import MockDate from 'mockdate';
 
+const systemTimeZone = dayjs().format('Z');
```

Comment:
Can we mock the timezone instead (set `dayjs` to use a certain predetermined timezone in the tests)?

This can make the tests more robust / independent from environment.

Reply:
I tried 3 ways, none of them worked:
1) With package for mock timezone. It conflicts with the other package `mockdate`.
2) Redefine `dayjs` package for jest with dayjs's method `.tz()` . `dayjs()` and `dayjs.tz()` work in different ways.
3) Set default timezone for jest with TZ env. variable. The problem is, that it doesn't work for windows. https://github.com/nodejs/node/issues/4230

Reply:
OK, let's leave it as it is for now.